### ISSUE TITLE
Offline mysterious stone calculation is wrong

### ISSUE THREAD

**Issue by Nomeru** [MelvorIdle/melvoridle.github.io#1159]
It seems offline calculation uses the formula getRhaelyxStoneChance() = 0.0001 (1/10k), rather than the .00025 (1/4000) it should be.

Online uses items[CONSTANTS.item.Mysterious_Stone].dropRate / 100 = 0.00025 so it's not affected it seems.

**Comment by Silberdrachi** [comment]
```
function getRhaelyxStoneChance() {
  if (itemStats[CONSTANTS.item.Crown_of_Rhaelyx].stats[0] > 0) return 0.0001;
   else return 0;
}
```

replaced with 

```
function getRhaelyxStoneChance() {
  if (itemStats[CONSTANTS.item.Crown_of_Rhaelyx].stats[0] > 0) return items[CONSTANTS.item.Mysterious_Stone].dropRate / 100;
   else return 0;
}
```

**Comment by coolrox95** [comment]
Fixed as of Beta v1.0.
